engine/src/main/java/com/arcadedb/query/sql/executor/TraverseExecutionPlanner.java

```diff
@@ -248,7 +248,7 @@ private void handleClustersAsTarget(final SelectExecutionPlan plan, final List<B
       }
 
       final FetchFromClusterExecutionStep step = new FetchFromClusterExecutionStep(bucketId, context, profilingEnabled);
-      // TODO: THIS SEEMS A BUG
+      // TODO: THIS SEEMS A BUG (maybe because if null is passed equals always returns false?)
```

Comment:
This code is the same in OrientDB: the support for the order by was never implemented in this class